keept.c

```diff
@@ -136,6 +136,7 @@ static void usage(const char * prgname, int more)
 #if HAVE_ABSTRACT_SOCKET_NAMESPACE
 		"     @: use socket in abstract namespace" nl
 #endif
+		"     S: remove socket at the end" nl
```

Comment:
tried to use lowercase flags only (although @ may require shift/alrgr etc, @ was just so good... ;D),
anyway perhaps 'u' -- like unlink instead ?

Reply:
i'm totally good with 'u' flag. btw 'S' was meant to be mnemonic for "socket" and uppercase usually means negation among cli options. '@' is really a good mnemonic.